mongodb/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gitpod/workspace-full:latest
+
+# Install mongodb
+RUN sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4
+RUN echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
+RUN sudo apt-get update \
+ && sudo apt-get install -y mongodb-org \
+ && sudo apt-get clean \
+ && sudo rm -rf /var/cache/apt/* /var/lib/apt/lists/* /tmp/*

```

Comment:
I think you can replace `sudo apt-get clean` and all these `rm -rf`s with just:

```bash
sudo rm -rf /var/lib/apt/lists/*
```

The others should already be cleaned up automatically by `apt-get`.

Reply:
Done